lib/modules/manager/gitlabci/extract.ts

```diff
@@ -31,6 +31,7 @@ import { getGitlabDep, replaceReferenceTags } from './utils';
 const componentReferenceRegex = regEx(
   /(?<fqdn>[^/]+)\/(?<projectPath>.+)\/(?:.+)@(?<specificVersion>.+)/,
 );
+const componentFQDNRegex = regEx(/[^/]+/);
```

Comment:
I don't understand why we need this simple regex? what do you check with that? isn't a simple string check enough?